tests/test_nblist.py

```diff
@@ -69,68 +69,68 @@ def test_neighborlist():
 
     for size in [35, 64, 129, 1025, 1259, 2029]:
 
-        nblist = custom_ops.Neighborlist_f64(size)
-        water_coords = get_water_coords(3, sort=False)
+        for precision, nblist in ((np.float32, custom_ops.Neighborlist_f32(size)), (np.float64, custom_ops.Neighborlist_f64(size))):
+            water_coords = precision(get_water_coords(3, sort=False))
```

Comment:
Is there a reason for casting the precision of the input coordinates? The input args always double

Reply:
Nope